src/views/pages/name/scottish-limited-partnership-name.njk

```diff
@@ -0,0 +1,107 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form class="form" action={{ props.currentUrl }} method="post">
+
+      <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+      {% include "includes/csrf_token.njk" %}
+
+      <h1 class="govuk-heading-xl">{{ i18n.scottishNamePage.title }}</h1>
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper">
+          <label class="govuk-label govuk-label--l" for="partnership_name">
+            {{ i18n.scottishNamePage.whatIsName }}
+          </label>
+        </h1>
+
+        {{ govukInput({
+          classes: "govuk-input--width-40",
+          errorMessage: errors.scottish_partnership_name if errors,
+          id: "partnership_name",
+          name: "partnership_name",
+          value: partnership_name,
```

Comment:
the value is supposed to be props.data.limitedPartnership.partnership_name

Reply:
changed value to match review: props.data.limitedPartnership.partnership_name